docs/robots.txt

```diff
@@ -0,0 +1,10 @@
+User-agent: *
+Disallow: /en/release-v0.4.x/
+Disallow: /en/release-v0.5.x/
+Disallow: /en/release-v0.6.x/
+Disallow: /en/release-v0.7.x/
+Disallow: /en/release-v0.8.x/
+Disallow: /en/release-v0.9.x/
+Disallow: /en/release-v0.10.x/
+Disallow: /en/release-v0.11.x/
+Disallow: /en/release-v0.12.x/
```

Comment:
robots.txt supports "glob" expressions, I guess this is an okay reference: https://developers.google.com/search/reference/robots_txt

So all of the above can be replaced with `Disallow: /*/release-v*/` - although I can't say how precise the expression is, it relies a bit on whether `*` matches `/` or not.

Alternatively, we can add just one `Allow: /*/latest/` and exclude everything else.

Reply:
Note to maybe add the sitemap 
https://kolibri.readthedocs.io/sitemap.xml

Example: https://docs.readthedocs.io/robots.txt